lib/rails_erd/domain/attribute.rb

```diff
@@ -88,6 +88,11 @@ def inheritance?
         @model.inheritance_column == name
       end
 
+      # Method allows false to be set as an attributes option when making custom graphs.
+      # Its functionality is implemented in the filtered_attributes method.
+      def false?
+      end
```

Comment:
We have no idea what these methods are for, just that its existence was required for the tests to pass. The logic in diagram.rb was sufficient to get it working.